mops-torch/src/sap.cpp

```diff
@@ -1,3 +1,8 @@
+#ifdef MOPS_CUDA_ENABLED
+#include <c10/cuda/CUDAGuard.h>
+#include <c10/cuda/CUDAStream.h>
+#endif
+
```

Comment:
I see that `mops-torch/src/sap.cpp` has not been modified past the headers (i.e. the stream is not actually taken into account), and the same is true for `opsaw` and `sasaw`. Is that correct?